ledger/partial/ptrie/node.go

```diff
@@ -21,7 +21,6 @@ type node struct {
 	lChild    *node           // Left Child
 	rChild    *node           // Right Child
 	height    int             // Height where the node is at
-	path      ledger.Path     // path
```

Comment:
Just checking if this wouldn't create any encoding/storage backward incompatibility. I don't think so as I believe VNs do not need to store partial tries.

Reply:
I believe PSMT are not stored. So it should be fine 🤞  😅